docs/guides/gb2025.md

```diff
@@ -45,9 +45,9 @@ The consequence of this setting is that only 71 cores per socket can be requeste
 !!! warning "Unable to allocate resources: Requested node configuration is not availabl"
```

Comment:
```suggestion
!!! warning "Unable to allocate resources: Requested node configuration is not available"
```

Reply:
Nice, one of many typos that I've missed in #57.